packages/abi-gen/test-cli/expected-output/python/abi_gen_dummy/__init__.py

```diff
@@ -54,9 +56,27 @@ class Tuple0xc9bdd2d5(TypedDict):
     accomplished via `str.encode("utf_8")`:code:
     """
 
-    innerStruct: Tuple0xcf8ad995
+    aField: int
+
+
+class Tuple0x1b9da225(TypedDict):
+    """Python representation of a tuple or struct.
```

Comment:
Definitely appreciate the usefulness for a detailed comment here.

Suggest some wording changes:


> Solidity compiler output does not include the names of structs that appear in method definitions. Instead, the ABI surfaces the variable name bound in scope. A tuple found in the ABI may have been written in Solidity as an unnamed tuple, or it may have been written as a named struct, but there is no way to tell from the compiler output. This class represents a tuple that appeared in a method definition...

Reply:
See https://github.com/0xProject/0x-monorepo/pull/2010/commits/a12a8a11e6a38d183f8b33b6d57f6ec3276a4dbd